README.md

```diff
@@ -123,36 +123,53 @@ each milestone.
 ### Next (5.0)
 
 * Improved and consistent visual style of built-in widgets.
-* Hands off approach to styling.  ipywidgets only provides an API for 
-  adding/removing CSS classes and the Style API.  The Style API provides the 
-  ability to set layout CSS traits in a mutable fashion. All other styling APIs 
+* Hands off approach to styling.  ipywidgets only provides an API for
+  adding/removing CSS classes and the Style API.  The Style API provides the
+  ability to set layout CSS traits in a mutable fashion. All other styling APIs
   are deprecated.
-* Ability to run ipywidgets outside of the notebook.
-* Initial refactoring to start to use npm for packaging the JavaScript code.
-* A new jupyter-js-widgets repository, containing all of the JS of ipywidgets.  
-  ipywidgets itself will no longer contain any JS.
-* A new interact API that behaves like Sage's equivalent.
+* Include a new interact API that behaves like Sage's equivalent.
+* Use npm for packaging the JavaScript code as `jupyter-js-widgets`.
+* Use the new notebook nbextension mechanism, introduced in Jupyter notebook 4.2
+  for the widgetsnbextension package, which will contain all of the notebook
+  specific widget Javascript.
+* Include two mechanisms for exporting widgets to static formats, rasterized
+  screencaptures of the widgets and an embed widget manager which is capable of
+  rendering widgets outside of the notebook.
+* Add examples for using the widgets outside of the notebook.
+* Add narrative docs describing the widgets basic usage and low level widget
+  spec information.
 
 ### Next+1 (6.0)
 
-* jupyter-js-widgets will be modernized, using a modern packaging specification,
-  a modern transpiler, and offloading much of JavaScript weight lifting to third 
-  party packages.
-* Explore how we can start to treat widgets as regular output, and not have to have
-  a separate widget area in the DOM.
-* Explore how the existing codebase will integrate with phosphor. In particular, explore
-  using phosphor properties to abstract away the details of backbone.js models and replace
-  the backbone models with phosphorjs widget-based views.
+* Unit tests for jupyter-js-widgets individual widgets
+* Unit tests for widgetsnbextension, especially manager.js
+* Transition to ES6 or TypeScript.
```

Comment:
Gradual transition or total ? This looks like Widgets **will be** TS or ES6, while you can mix versions, and gradually improve. Also precise that user won't need to write in es6 or TS.